game-core/src/main/java/games/strategy/engine/config/lobby/TestLobbyPropertyReaders.java

```diff
@@ -7,6 +7,8 @@
 /**
  * Collection of {@link LobbyPropertyReader} instances suitable for various testing scenarios.
  */
+// TODO: This class is intended only for use by test code. It is temporarily located in a production package while lobby
+// server code is being gradually moved to the lobby project.
 public final class TestLobbyPropertyReaders {
```

Comment:
I moved this test-specific class to a production package so it is available to the `lobby` project.  This is only temporary until the remaining lobby server types are moved to the `lobby` project.  At that time, this class will also move to the `lobby` project.

`MemoryPropertyReader` was also moved to a production package for the same reason.  It's not so much a test-specific class, but it is currently only used by test code (hence, why it is in the test source set).  However, this class is a bit more delicate because it is also used by non-lobby unit tests within the `game-core` project.  To move it back to the test source set may require splitting `game-core` into two projects: one for API and one for implementation.  That would allow us to create a test-specific project for the API (of which `PropertyReader` is) that could be used as a dependency by both the `game-core` implementation project and the `lobby` project.

For the time being, I think leaving `MemoryPropertyReader` in the production code is acceptable.

Reply:
For the bigger goal 👍